makefile

```diff
@@ -430,7 +430,10 @@ endif
 	@cp ${ECMD_ROOT}/ecmd-core/ext/cip/cmd/help/cipinstruct.htxt ${INSTALL_PATH}/help/.
 	@cp ${ECMD_ROOT}/ecmd-core/ext/cip/cmd/help/cipgetmemproc.htxt ${INSTALL_PATH}/help/.
 	@cp ${ECMD_ROOT}/ecmd-core/ext/cip/cmd/help/cipputmemproc.htxt ${INSTALL_PATH}/help/.
-	@cp ${ECMD_ROOT}/out_${TARGET_ARCH}/bin/ecmd.htxt ${INSTALL_PATH}/help/.
+	@cp ${ECMD_ROOT}/ecmd-core/cmd/help/ecmd.htxt ${INSTALL_PATH}/help/.
+ifeq (${EDBG_ISTEP_CONTROL}, yes)
```

Comment:
Right now i am hard coding p10 path. will change that in next PR to take chip=p10/p11 argument as when when configuring